addons/xm157/dev/generate_reticle.py

```diff
@@ -70,13 +66,13 @@
         (image_center + 20 * pixels_per_mrad, image_center - 2 * pixels_per_mrad),
         (image_size, image_center - 2 * pixels_per_mrad),
     ], fill=(0, 0, 0))
-    d.polygon([
-        (image_center - 2 * pixels_per_mrad, image_size),
-        (image_center - 2 * pixels_per_mrad, image_center + 20 * pixels_per_mrad),
-        (image_center, image_center + 15 * pixels_per_mrad),
-        (image_center + 2 * pixels_per_mrad, image_center + 20 * pixels_per_mrad),
-        (image_center + 2 * pixels_per_mrad, image_size),
-    ], fill=(0, 0, 0))
+    # d.polygon([
+    #     (image_center - 2 * pixels_per_mrad, image_size),
+    #     (image_center - 2 * pixels_per_mrad, image_center + 20 * pixels_per_mrad),
+    #     (image_center, image_center + 15 * pixels_per_mrad),
+    #     (image_center + 2 * pixels_per_mrad, image_center + 20 * pixels_per_mrad),
+    #     (image_center + 2 * pixels_per_mrad, image_size),
+    # ], fill=(0, 0, 0))
```

Comment:
yeah, this removes the large vertical "arrow" bellow the dots, which seems to more closely match the real one
I'll clean it up
```suggestion
```